permissions_validators/src/public_blockchain/mod.rs

```diff
@@ -44,6 +44,8 @@ pub enum PredefinedPermissionToken {
     TransferOnlyFixedNumberOfTimesPerPeriod(transfer::CanTransferOnlyFixedNumberOfTimesPerPeriod),
     /// Can un-register asset with the corresponding asset definition.
     UnregisterAssetWithDefinition(unregister::CanUnregisterAssetWithDefinition),
+    /// Can change configuration parameters.
+    CanChangeConfigParameters(set_parameter::CanChangeConfigParameters),
```

Comment:
We are currently ignoring `NewParameter` instruction inside `ChangeParametersOnlyForSignerAccount` and `GrantedChangeConfigParameters` validators. 
But since parameters currently doesn't affect anything + we are moving towards runtime validators, this is not something we want to fix right now imo.
I will create ticket to keep track of this issue.